CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 - Fix automatically login to last remembered account
 - Fix scanning qr code and general improvement of the whole process
 - Fix avatar initials of the text avatars, of people with no displayname, inside of the messagelist
+- Fix search shows placeholders
```

Comment:
What does this mean?

Reply:
#1762
can you think of a better sentence?